src/init/init.c

```diff
@@ -20,7 +43,15 @@ t_app	*init_shell(char *envp[])
 	if (!app)
 		return (NULL);
 	app->malloc_list = NULL;
-	app->envp = envp;
+	app->envp = copy_envp(envp, &app->malloc_list);
+/*	
+	int i = 0;
+	while (app->envp[i])
+	{
+		ft_printf("%s\n", app->envp[i]);
+		i++;
+	}
+*/	
```

Comment:
same as the comment above. let's try not to mix commits from different features. sometimes it's inevitable but I imagine here it was an accident?